client/my-sites/themes/test/logged-out.jsx

```diff
@@ -95,6 +95,15 @@ describe( 'logged-out', () => {
 		} );
 
 		it( 'renders without error when no themes are present', () => {
+			this.store.dispatch( receiveThemes( [], 'wpcom' ) );
+			this.store.dispatch( {
+				type: THEMES_REQUEST_SUCCESS,
+				siteId: 'wpcom',
+				query: DEFAULT_THEME_QUERY,
+				found: 0,
+				themes: []
+			} );
```

Comment:
You should be able to drop the second `dispatch` and add the additional parameters to `receiveThemes` instead since it now actually [invokes `THEMES_REQUEST_SUCCESS` for us](https://github.com/Automattic/wp-calypso/blob/caa9d9fcf5374945a1c37a44887a12301389fcf5/client/state/themes/actions.js#L130-L136) 😄